src/config/paper/spigot.yml

```diff
@@ -85,198 +174,180 @@ messages:
-    arrow-despawn-rate:
-      default: "1200"
-      description: The number of ticks before an arrow despawns.
     below-zero-generation-in-existing-chunks:
       default: "true"
       description: >-
         Whether to allow conversion of existing chunks to full (post 1.18)
         height.
-    dragon-death-sound-radius:
-      default: "0"
-      description: >-
-        The number of blocks that the dragon death sound is audible. Set to 0
-        for global (including cross-dimension).
+    merge-radius:
+      exp:
+        default: "-1"
+        description: >-
+          The range, in blocks, that exp orbs will combine at on initial spawn.
+          This behavior is not present in Vanilla and doesn't impact the usual
+          merge range once spawned. Set to 0 or less to disable.
+      item:
+        default: "0.5"
+        description: "The range, in blocks, that items will combine within."
+    ticks-per:
+      hopper-check:
+        default: "1"
+        description: The ticks between checks to pull items.
+      hopper-transfer:
+        default: "8"
+        description: The ticks between hopper item movements.
+    hopper-amount:
+      default: "1"
+      description: "How many items a hopper should move at a time, limited to stack size."
+    hopper-can-load-chunks:
+      default: "false"
+      description: Whether to prevent hoppers from loading chunks.
+    hunger:
+      combat-exhaustion:
+        default: "0.1"
+        description: How much exhaustion to give from attacking.
+      jump-sprint-exhaustion:
+        default: "0.2"
+        description: How much exhaustion to give from sprint jumping.
+      jump-walk-exhaustion:
+        default: "0.05"
+        description: How much exhaustion to give from jump walking.
+      other-multiplier:
+        default: "0.0"
+        description: The exhaustion multiplier for normal walking and crouching.
+      regen-exhaustion:
+        default: "6.0"
+        description: How much exhaustion to give from regenerating health.
+      sprint-multiplier:
+        default: "0.1"
+        description: The exhaustion multiplier for sprinting on the ground.
+      swim-multiplier:
+        default: "0.01"
+        description: >-
+          The exhaustion multiplier for when "swimming". This occurs when the
+          player is:
+
+          - actually swimming
+
+          - walking underwater (eyes below surface)
+
+          - walking on water (eyes above surface)
     enable-zombie-pigmen-portal-spawns:
       default: "true"
       description: >-
         Whether to allow zombified piglins to spawn inside nether portals. This
         setting does not affect portal travel for any mobs.
+    zombie-aggressive-towards-villager:
+      default: "true"
+      description: Whether zombies try to seek out and attack villagers.
+    dragon-death-sound-radius:
```

Comment:
Maybe note what happens in relation to the new gamerule (applies to the other similar option)